from0k2bp.tex

```diff
@@ -959,85 +977,81 @@
 
 The Prover's job will now be to convince the verifier that
 \emph{Pedersen commitments }to these three quantities obey $z = \mathbf{x} \cdot \mathbf{y}$; so we
-assume upfront that the three commitments are known, we'll call them
-from now $C_z, C_x, C_y$:
-\begin{align*}
-& C_z = tH + zG \\
-& C_x = rH + \mathbf{xG} \\
-& C_y = sH + \mathbf{yG} \\
-\end{align*}
-(remember our notation cheat: the bolded parts are actually summations).
+assume upfront that the three commitments are known, 
+we'll denote them $C_z, C_x$ and $C_y$ from now on:
+\begin{align}
+C_z &:= tH + zG \label{inner_product_proof_C_xyz_def:1}\\
+C_x &:= rH + \mathbf{xG} \label{inner_product_proof_C_xyz_def:2}\\
+C_y &:= sH + \mathbf{yG} \label{inner_product_proof_C_xyz_def:3}
+\end{align}
+(Remember our notation cheat\footnote{Specifically definition \eqref{vector-pedersen-commitment-def:1}}: 
```

Comment:
No, it's not definition 1. It's the last bullet point in Section 1.5 "Notation".

Reply:
I have rather removed the footnote. It's not good.